src/utils/scaffold.ts

```diff
@@ -68,6 +69,18 @@ function modifyMainTs(pathToMainTs: string) {
   }
 }
 
+function modifyTsconfigBuildJson(pathToTsconfigBuildJson: string) {
+  const TsconfigBuildJson = fs.readFileSync(pathToTsconfigBuildJson, { encoding: 'utf8' });
```

Comment:
I would do it differently. Use JSON.parse or similar to parse the content. Then check if the exclude property is present and either extend the existing one (the spread operator ... is good for that) or set it.

If you do it this way you do not need the if check below, because the modification will work. You can handle writeFileSync failures though.